src/Tgstation.Server.Host.Console/tgs.service

```
@@ -0,0 +1,11 @@
+[Unit]
+Description=tgstation-server
+After=network.target
+After=mariadb.target
+
+[Service]
+ExecStart=bash /opt/tgs/tgs.sh
```

```suggestion
ExecStart=/bin/bash /opt/tgs/tgs.sh
```

Always use explicit paths.